tests/PHPStan/Rules/Methods/ReturnTypeRuleTest.php

```diff
@@ -809,4 +809,15 @@ public function testBug8223(): void
 		]);
 	}
 
+	public function testBug8146b(): void
```

Comment:
nit: I misstyped the testname

```suggestion
	public function testBug8146c(): void
```